Cabal/src/Distribution/Simple/Flag.hs

```diff
@@ -124,6 +125,12 @@ maybeToFlag :: Maybe a -> Flag a
 maybeToFlag Nothing  = NoFlag
 maybeToFlag (Just x) = Flag x
 
+maybeStringToBoolFlag :: Maybe String -> Flag Bool
+maybeStringToBoolFlag Nothing = NoFlag
```

Comment:
Can it be defined simply as `maybeToFlag . (readMaybe =<<)`?

Reply:
@ulysses4ever I changed it. Thanks, I hadn't used that function before.